taxi_service/urls.py

```diff
@@ -13,9 +13,13 @@
     1. Import the include() function: from django.urls import include, path
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
+
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
+from django.conf import settings
+from django.conf.urls.static import static
 
-urlpatterns = [
-    path("admin/", admin.site.urls),
-]
+urlpatterns = [path("admin/", admin.site.urls),
+               path("", include("taxi.urls", namespace="taxi"))
+               ] + static(settings.STATIC_URL,
+                          document_root=settings.STATIC_ROOT)
```

Comment:
plese format this 

you can use `black`

Reply:
done